recipes/fakeit/all/conanfile.py

```diff
@@ -37,16 +48,23 @@ def requirements(self):
             raise ConanInvalidConfiguration("%s is not (yet) available on cci" % self.options.integration)
 
     def package_id(self):
-        self.info.header_only()
+        self.info.clear()
 
     def validate(self):
-        minimal_cpp_standard = "11"
         if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, minimal_cpp_standard)
+            check_min_cppstd(self, self._min_cppstd)
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def build(self):
+        apply_conandata_patches(self)
```

Comment:
This is generally not allowed but since there's no side effects let's try it.

Please refrain from adding dead code though, it's likely it will be removed